MockerTests/MockerTests.swift

```diff
@@ -42,11 +42,11 @@ final class MockerTests: XCTestCase {
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: MockerTests/MockerTests.swift;
  Line: 42;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)